ktor-netty/src/org/jetbrains/ktor/netty/NettyApplicationRequest.kt

```diff
@@ -28,6 +25,10 @@ class NettyApplicationRequest(val request: FullHttpRequest) : ApplicationRequest
             QueryStringDecoder(request.uri).parameters().forEach {
                 appendAll(it.key, it.value)
             }
+            // as far as we have full request we can access request body many times
+            if (contentType().contentType == "application" && contentType().contentSubtype == "x-www-form-urlencoded") {
+                appendAll(parseUrlEncodedParameters())
```

Comment:
yes, we can add it but still can't compare contentType because it could contain "parameters" that prevent it to be equal to our constant